sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultClient.java

```diff
@@ -96,6 +110,28 @@
             keyVaultUri = keyVaultUri + "/";
         }
         this.keyVaultUrl = keyVaultUri;
+        String dnsSuffix = Optional.of(keyVaultUri)
```

Comment:
`dns` is short for `Domain Name System`. So I prefer `domainNameSuffix` than `dnsSuffix`.